applications/HDF5Application/python_scripts/multiple_mesh_temporal_output_process.py

```diff
@@ -73,7 +73,7 @@ def Factory(settings, Model):
 
 
 def MultipleMeshTemporalOutputProcessFactory(core_settings, Model):
-    return core.Factory(core_settings, Model)
+    return core.Factory(core_settings, Model, KratosMultiphysics.OutputProcess)
```

Comment:
in the core settings, is it possible to do something like this?
```patch
-            "process_step": "finalize_solution_step",
+            "process_step": "print_output",
```
I don't know what consequences this may have for the existing code

Reply:
Yes it's possible and [implemented](https://github.com/KratosMultiphysics/Kratos/blob/9cca10ed5700db570764da6a2c7adc53a8cdf41c/applications/HDF5Application/python_scripts/core/__init__.py#L60) (if you mean a process performs an output on each call, without checks), but it's called ```"output"``` instead of ```"print_output"```.